src/js/user-profile/containers/UserProfileApp.jsx

```diff
@@ -1,40 +1,84 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+import ProgressButton from '../../common/components/form-elements/ProgressButton';
+import Modal from '../../common/components/Modal';
+
+import { removeSavedForm } from '../actions/index';
 import UserDataSection from '../components/UserDataSection';
 import AuthApplicationSection from '../components/AuthApplicationSection';
-
+import FormList from '../components/FormList';
 import RequiredLoginView from '../../common/components/RequiredLoginView';
 
+import { isSIPEnabledForm } from '../helpers';
+
 class UserProfileApp extends React.Component {
-  render() {
-    const view = (
-      <div className="row user-profile-row">
-        <div className="usa-width-two-thirds medium-8 small-12 columns">
-          <h1>Your Account</h1>
-          <div>
-            <AuthApplicationSection
-              userProfile={this.props.profile}
-              verifyUrl={this.props.verifyUrl}/>
-            <UserDataSection/>
-          </div>
-        </div>
-      </div>
-    );
-
-    return (
-      <div>
-        <RequiredLoginView
-          authRequired={1}
-          serviceRequired="user-profile"
-          userProfile={this.props.profile}
-          loginUrl={this.props.loginUrl}
-          verifyUrl={this.props.verifyUrl}>
-          {view}
-        </RequiredLoginView>
-      </div>
-    );
+  constructor(props) {
+    super(props);
+    this.state = { modalOpen: false };
   }
+
+ removeForm = () => {
+   this.toggleModal();
+   this.props.removeSavedForm(this.state.formId);
+ }
+
+ toggleModal = (formId) => {
+   this.setState({ formId, modalOpen: !this.state.modalOpen });
```

Comment:
You should be able to leave out `formId` here.

Reply:
Really? How should `removeForm` be fed the `formId`?

Reply:
I mean that I don't think this `setState` call will erase `formId` if you leave it out here. I could be wrong, though.